core/App/screens/ListCredentials.tsx

```diff
@@ -28,21 +25,22 @@ const ListCredentials: React.FC = () => {
       style={{ backgroundColor: ColorPallet.brand.primaryBackground }}
       data={credentials.sort((a, b) => new Date(b.createdAt).valueOf() - new Date(a.createdAt).valueOf())}
       keyExtractor={(credential) => credential.id}
-      renderItem={({ item: credential, index }) => (
-        <View
-          style={{
-            marginHorizontal: 15,
-            marginTop: 15,
-            marginBottom: index === credentials.length - 1 ? 45 : 0,
-          }}
-        >
-          <CredentialCard
-            credential={credential}
-            revoked={revoked.has(credential.id)}
-            onPress={() => navigation.navigate(Screens.CredentialDetails, { credentialId: credential.id })}
-          />
-        </View>
-      )}
+      renderItem={({ item: credential, index }) => {
+        return (
```

Comment:
is this change needed? seems like the template is just being wrapped in a `return (...)`. I don't mind either way, just curious

Reply:
Probably just carry over from testing